rpcserver.go

```diff
@@ -975,6 +975,34 @@ func (r *rpcServer) ListAssets(ctx context.Context,
 		return nil, err
 	}
 
+	var (
+		filteredAssets   []*taprpc.Asset
+		unconfirmedMints uint64
+	)
+
+	// We now count and filter the assets according to the
+	// IncludeUnconfirmedMints flag.
+	for idx := range rpcAssets {
+		switch {
+		// If the asset isn't confirmed yet, we count it but only
+		// include it in the output list if the client requested it.
+		case rpcAssets[idx].ChainAnchor.BlockHeight == 0:
```

Comment:
Any particular reason we're opting to do this here instead of within the db query? Mainly just for a smaller diff?

Reply:
Mainly so we can _count_ how many assets we filter out. Which is purely a user experience thing, as the user would get a hint that their freshly minted assets might not appear in the list because they're unconfirmed.

I guess for now this works and avoids us needing to do two queries. But once we add pagination, we might need to change the approach here... Can do it now if you think it's worth doing.
Kinda forgot about our plans to add pagination, so happy to change it.